pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/reports/MetadataVerifyContent.kt

```diff
@@ -6,10 +6,10 @@ import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 class MetadataVerifyContent : SchemaDefinition {
 
     @GraphQLDescription("Schema name for the report content")
-    override val schemaName: String = "dex-metadata-verify"
+    override val schemaName: String = "upload-status"
```

Comment:
Why was this changed?  This should still be the metadata verify one.

Reply:
This change is reverted for the Metadata classes. 
However, upon closer inspection, noticed that the default SchemaDefinition values should be set to the following:
_companion object {
        val schemaDefinition = SchemaDefinition(schemaName = "upload-status", schemaVersion = "1.0.0")
    }_

The content_schema_name and content_schema_version for the upload is updated to "upload-status", "1.0.0" respectively in the reports.py scripts